shared/settings/notifications/push-prompt.native.tsx

```diff
@@ -83,13 +83,8 @@ const styles = Styles.styleSheetCreate(
         color: Styles.globalColors.white,
       },
       image: Styles.platformStyles({
-        common: {
```

Comment:
Doesn't this change phones

Reply:
@chrisnojima yes, phone was bad too, there are before and after screens for phone above